pkg/analytics/config_test.go

```diff
@@ -50,7 +50,7 @@ func Test_Get(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			dir := t.TempDir()
 
-			os.Setenv(model.OktetoFolderEnvVar, dir)
+			os.Setenv(constants.OktetoFolderEnvVar, dir)
```

Comment:
this is chaning the Env when running the test, should we use t.SetEnv or defer the restoring of the original env?

Reply:
I'll address this one here because it's really quick, but I think they are outside the scope of the pr